install.sh

```diff
@@ -12,9 +12,12 @@ cat NOTICE
 echo
 
 license="${HAB_LICENSE:-}"
+export response
```

Comment:
Nit: It would be safer to use `declare` here. `export` will make the `response` variable available to processes spawned by this script, which could cause undesired and unexpected behavior.  We're protected here since we use `sudo` to call the remaining scripts and we don't ever call the `s3migrate.sh` script from here, where `response` is also used. 

I don't feel this needs to change, but given the amount of shell we have in this project felt it was still worth mentioning.

Reply:
Excellent catch! I think fixing this is the right thing.